release-doc-in-process/all_vf.TTL

```diff
@@ -71,6 +71,10 @@ vf:DistributionAgreement  a       owl:Class ;
         rdfs:subClassOf   vf:Agreement ;
         rdfs:comment      "An agreement to distribute some resource in exchange for inputs or outputs created by other agents." .
 
+vf:ExchangeRate a owl:Class ;
+        rdfs:label        "vf:ExchangeAgreement" ;
```

Comment:
label copy/paste?

Reply:
I think we did it because WebVOWL uses those labels on diagrams

Reply:
but do you want to use ExchangeAgreement for ExchangeRate?